pkg/connector/projects.go

```diff
@@ -203,11 +214,6 @@ func (o *projectBuilder) Revoke(ctx context.Context, grant *v2.Grant) (annotatio
 
 func projectResource(project *client.Project, parentResourceID *v2.ResourceId, isOnPremise bool) (*v2.Resource, error) {
 	var annotations []proto.Message
-	if !isOnPremise {
```

Comment:
project is a child resource type of group, so getting all the group members should get all users